packages/cli/src/lib/commands/hub.ts

```diff
@@ -56,19 +58,62 @@ export const hub: CommandDefinition = (program) => {
 
                 displayObject(hosts, profileConfig.format);
             });
-    }
-
-    if (isProductionEnv(profileConfig.env)) {
         hubCmd
             .command("info")
             .description("Display info about the default Hub")
             .action(async () => {
-                const { lastSpaceId: space, lastHubId: id } = sessionConfig.get();
-                const managerClient = getMiddlewareClient().getManagerClient(space);
-                const hosts = await managerClient.getHosts();
-                const host = hosts.find((h: any) => h.id === id);
+                displayObject((await getInfo()).host, profileConfig.format);
+            });
+        hubCmd
+            .command("disconnect")
+            .description("Disconnect self hosted Hubs from space")
+            .argument("<space_name>", "The name of the Space")
+            .option("--id <id>", "Hub Id")
+            .option("--all", "Disconnects all self-hosted Hubs connected to Space", false)
+            .action(async (spaceName: string, options: { id: string, all: boolean }) => {
+                const managerClient = mwClient.getManagerClient(spaceName);
+                let opts = { } as PostDisconnectPayload;
+
+                if (typeof options.id === "string") {
```

Comment:
id should be uuid, add validation for this type, (validator is ready in uuid package)